lib/resque/scheduler/env.rb

```diff
@@ -56,32 +56,22 @@ def setup_pid_file
 
       def setup_scheduler_configuration
         Resque::Scheduler.configure do |c|
-          if options.key?(:app_name)
```

Comment:
Maybe something like this?

~~~ diff
       def options
         @options ||= {}.tap do |o|
-           CLI_OPTIONS_ENV_MAPPING.map { |key, envvar| o[key] = env[envvar] }
+           CLI_OPTIONS_ENV_MAPPING.map { |key, envvar| o[key] = env[envvar] if env.include?(envvar) }
         end
       end
~~~